internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -53,6 +64,79 @@ func (s *Server) SetUpSession(stream s2av2pb.S2AService_SetUpSessionServer) erro
 	}
 }
 
+// TODO(rmehta19): consider also returning an error with this function.
```

Comment:
Is this TODO needed? Aren't we already returning an error?

Reply:
Yes we are, I must've missed deleting this line, thanks for catching this!